libs/api/domains/driving-license/src/lib/drivingLicense.type.ts

```diff
@@ -78,6 +83,7 @@ export enum RequirementKey {
   drivingAssessmentMissing = 'DrivingAssessmentMissing',
   drivingSchoolMissing = 'DrivingSchoolMissing',
   deniedByService = 'DeniedByService',
+  localResidency = 'localResidency',
```

Comment:
Should this not be `localResidency = 'LocalResidency'`?

I'm seeing things like
```
case RequirementKey.LocalResidency:
```
elsewhere, or is this unrelated maybe?